api/API_MANIFESTO.adoc

```diff
@@ -12,20 +12,20 @@ Examples:
  
  * Tasks, concurrency, counters uses cases
 
-The dependencies to thirty-party libraries must be avoided. This module should contain dependencies to JDK classes, 
-not more.
+The dependencies to thirty-party libraries must be avoided. This module should contain dependencies to JDK classes, not more.
```

Comment:
I think the wording needs updating, something like:

"This module should only contain dependencies on JDK classes and dependencies on third-party libraries must be avoided."

Reply:
Done